var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -21,7 +21,6 @@ class Flecsi(CMakePackage):
     git      = 'https://github.com/laristra/flecsi.git'
 
     version('develop', branch='master', submodules=False, preferred=True)
```

Comment:
This version can actually be called `master` now that #1983 has been merged.

Reply:
@junghans Do we want to change that in the FleCSI package? develop=>master

Reply:
Meh